lib/client.ts

```diff
@@ -132,8 +133,7 @@ class ClientManager extends EventTarget {
   }
 
   private async _getConfig(ozoneDid?: string) {
-    const builtIn =
-      ozoneDid || process.env.NEXT_PUBLIC_OZONE_SERVICE_DID || undefined
+    const builtIn = ozoneDid || OZONE_SERVICE_DID
     return await getConfig(builtIn)
```

Comment:
For the Ozone distribution we need to leave the possibility open that `builtIn` is not set, and may be determined at runtime within `getConfig`.

Reply:
yep, cleaning up the fallback of service did to our labeler should be enough for that, no?